### PR TITLE
Fixes #92 MoBi portable

### LINKED ISSUE #92: MoBi Portable
See discussion in forum
Open-Systems-Pharmacology/Forum#9

and some details about similar PK-Sim feature 
https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/161

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@rwmcintosh Very similar to what was just implemented in PK-Sim.

File: src/MoBi.UI/Services/ApplicationStartup.cs [hunk 1]
```diff
@@ -235,8 +235,8 @@ public void InitializeForStartup()
          //Register log4Net factory and set the path to configuration file
          var log4NetLogFactory = new Log4NetLogFactory();
 
-         log4NetLogFactory.Configure(new FileInfo(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "log4net.config.xml")));
-         log4NetLogFactory.UpdateLogFileLocation(config.AllUsersFolderPath);
+         log4NetLogFactory.Configure(new FileInfo(config.LogConfigurationFile));
+//         log4NetLogFactory.UpdateLogFileLocation(config.AllUsersFolderPath);
```
Comment (rwmcintosh) [inline src/MoBi.UI/Services/ApplicationStartup.cs]: Do you need to keep that around?

**Comment by rwmcintosh** [comment]
Just do something or nothing with that line that's commented out in ApplicationStartup.cs and merge.

**Comment by msevestre** [comment]
Updated

**Pull request merged**